src/init.ts

```diff
@@ -0,0 +1,64 @@
+import { getLogger } from './tools/logger';
+import * as util from './tools/util';
+
+const logger = getLogger(__filename);
+
+export async function initialize() {
```

Comment:
❗ 
The new initialization logic tries to create/enroll the ADMIN and SYSTEM users in case their wallets do not exist.
This is a slightly updated version of the one that is in the integration-gateway repo